frontend/fingertips-frontend/components/organisms/BasicTable/convertBasicTableToCsvData.ts

```diff
@@ -0,0 +1,37 @@
+import { CsvData, CsvRow } from '@/lib/downloadHelpers/convertToCsv';
+import { BasicTableData } from '.';
+import { CsvHeader } from '@/components/molecules/Export/export.types';
+import { HealthDataPointTrendEnum } from '@/generated-sources/ft-api-client';
+
+export const convertBasicTableToCsvData = (
+  indicatorData: BasicTableData[]
+): CsvData => {
+  const header: CsvRow = [
+    CsvHeader.IndicatorId,
+    CsvHeader.IndicatorName,
+    CsvHeader.Period,
+    CsvHeader.Count,
+    CsvHeader.ValueUnit,
+    CsvHeader.Value,
+    CsvHeader.RecentTrend,
+  ];
```

Comment:
```suggestion
  const header: CsvRow = [
    CsvHeader.IndicatorId,
    CsvHeader.IndicatorName,
    CsvHeader.Period,
    CsvHeader.Area,
    CsvHeader.AreaCode,
    CsvHeader.RecentTrend,
    CsvHeader.Count,
    CsvHeader.ValueUnit,
    CsvHeader.Value,
  ];
```

Whilst not consistent with the visual table presented on screen the above changes to the columns makes it more consistent with the other CSV exports. In a similar fashion although the area and code are always England, please add them anyway for consistency with other exports.